packages/sdks/javascript/tests/Exceptions.spec.ts

```diff
@@ -79,3 +82,15 @@ test("request network error should reject with a NetworkError", async () => {
     expect(e).toBeInstanceOf(NetworkError)
   }
 });
+
+test("Expired access_token should trigger callback function", async () => {
+  const mockCallback = jest.fn(x => x);
+
+  const options = { callbacks: { onTokenExpired: mockCallback } } as unknown as ConfigurationOptions
+  apiClient = new DevopnessApiClient(options);
+
+  reqMock.onGet('/users/me').replyOnce(401, {});
```

Comment:
I think this test would be better to understand by explicitly passing an expired token in the request.

So, the expected behaviour, is that by returning from the login an `accessToken` with a short `expiresIn`, the expiration validation at 

https://github.com/devopness/devopness/blob/e6797f9f649355560ac52665f8ba280128076e8d/packages/sdks/javascript/src/services/ApiBaseService.ts#L137-L139

would fail. Therefore, the better written test would look like:

```jsx
test("Expired access_token should trigger callback function", async () => {
  const email = 'test@test.com'
  const password = 'testpassword'
  const mockCallback = jest.fn(x => x);
  const options = { callbacks: { onTokenExpired: mockCallback } } as unknown as ConfigurationOptions
  
  apiClient = new DevopnessApiClient(options);

  reqMock.onPost('/users/login').replyOnce(200, {
    'access_token': '<access_token>',
    'refresh_token': '<refresh_token>',
    'expires_in': -10 // suppose the token is already expired
  });
  reqMock.onGet('/users/me').replyOnce(401, {}); // replied with a 401 because the access_token is expired.

  await apiClient.users.loginUser({ email, password })
  await apiClient.users.getUserMe()

  expect(mockCallback.mock.calls).toHaveLength(1)
});
```